test/describe_context.rb

```diff
@@ -0,0 +1,9 @@
+BDDSM.describe 'Test case with context' do
+  context 'Context example 1' do
```

Comment:
Is it possible to have nested contexts?

Reply:
Yes. I added one example to tests few seconds ago.